hexrd/material.py

```diff
@@ -69,48 +75,111 @@ class Material(object):
 
     The class references materials by name and contains lattice and
     space group data.
+    default data is for nickel, but name is material
     """
-    DFLT_NAME = 'material'
-    DFLT_SGNUM = 230
-    DFLT_LPARMS = [_angstroms(1.0), _angstroms(1.0), _angstroms(1.0),
+    DFLT_NAME = 'material.xtal'
+    DFLT_XTAL = 'Ni'
+    DFLT_SGNUM = 225
+    '''
+    some materials have more than one space group setting. for ex
+    the diamond cubic system has two settings with the origin either
+    at (0,0,0) or at (1/4,1/4,1/4) etc. this handle takes care of these
+    cases. but the defaiult is always 1
+    '''
+    DFLT_SGSETTING = 1
+
+    DFLT_LPARMS = [_angstroms(3.61), _angstroms(3.61), _angstroms(3.61),
                    _degrees(90.0), _degrees(90.0), _degrees(90.0)]
-    DFLT_SSMAX = 50
+    DFLT_SSMAX = 100
 
     DFLT_KEV = valWUnit('wavelength', 'energy', 80.725e0, 'keV')
     DFLT_STR = 0.0025
     DFLT_TTH = numpy.radians(0.25)
-    DFLT_ATOMINFO = numpy.array([[0, 0, 0, 1]])
-    """Fractional Atom Position of an atom in the unit cell followed by the
-    number of electrons within that atom. The max number of electrons is 96.
+    DFLT_TTHMAX = numpy.radians(160.0)
     """
+    ATOMINFO    Fractional Atom Position of an atom in the unit cell followed by the
+    site occupany and debye waller (B) factor in nm^(-2)
+
+    ATOMTYPE    atomic number of all the different species in the unitcell
+    """
+    DFLT_ATOMINFO = numpy.array([[0, 0, 0, 1, 0.033]])    
+    DFLT_ATOMTYPE = numpy.array([28])
+
+    '''
+    the dmin parameter is used to figure out the maximum sampling for g-vectors
+    this parameter is in angstroms
+    '''
+    DFLT_DMIN = _angstroms(0.2)
 
-    def __init__(self, name=DFLT_NAME, cfgP=None):
+    '''
+    default space group setting
+    '''
+    DFLT_SGSETTING = 0
```

Comment:
This is getting set twice. Once here, and once on [line 89](https://github.com/HEXRD/hexrd/pull/55/files#diff-9e92e5bccd03281b17dc651f596ca74bR89).

Reply:
default should be zero. removing the first one in the next commit.